### PR TITLE
Improve logging. Fixes #13

### LINKED ISSUE #13: Can we get better logging of infected files?
If I have the log level set to everything, I get these messages:

```
ws-prod-filehost-01 ownCloud[22621]: {files_antivirus} Exec scan: /trytwo.exe
Jan 21 11:17:25 ws-prod-filehost-01 ownCloud[22621]: {files_antivirus} Virus(es) found: Eicar-Test-Signature
Jan 21 11:17:26 ws-prod-filehost-01 ownCloud[22621]: {files_antivirus} Email: me@blah.tld
```

If I set it to "Warnings, errors, and fatal issues", I only get:

```
Jan 21 11:17:25 ws-prod-filehost-01 ownCloud[22621]: {files_antivirus} Virus(es) found: Eicar-Test-Signature
```

That isn't very helpful. It doesn't say what file is infected, or what account the file belongs to.

An imaginary situation would be something like a user doesn't receive the email OwnCloud sent telling them that the file was infected (say postfix died for some reason), so they call me up and complain. I get their account information and the file name, then go looking through my logs. Ideally, I'd like to be able to see a log message that says "Virus(es) found: Eicar-Test-Signature. Account: username. File: /blah/infected.exe".

Can we get a message like that? 

What log level would be best for it? I, personally, believe that an infected file is something we ALWAYS want to know about. Currently, that would mean a fatal issue. Though, since it doesn't stop execution of the application, I'm not sure it really qualifies...

### PR DESCRIPTION
**Pull request by VicDeo**
Log owner and path if file is infected

### CODE REVIEW

**Comment by karlitschek** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
Log message looks good :+1:

**Comment by VicDeo** [comment]
@owncloud-bot retest this please

**Comment by DeepDiver1975** [comment]
> @owncloud-bot retest this please

this does not work here - you have to go to the travis site and click restart on the build

File: lib/scanner.php [hunk 1]
```diff
@@ -75,36 +81,59 @@ public static function av_scan($path) {
...
+				return;
+			}
+			
+			$fileStatus = self::scanFile($filesView, $path);
+			$result = $fileStatus->getNumericStatus();
+			$account = \OCP\User::getUser();
+			if (!$account){
+				$account = 'Guest';
+			}
+			$result = Status::SCANRESULT_INFECTED;
```
Comment (VicDeo) [inline lib/scanner.php]: @DeepDiver1975 it seems my eyes were somewhere else when I was looking here ;)

**Pull request merged**